### ISSUE TITLE
Enable point_in_time_restore_containers for storage account

### ISSUE THREAD

**Issue by kumehta** [hashicorp/terraform-provider-azurerm#13286]
Hi Team,
I am using azurerm version 2.75.0

<img width="577" alt="point_in_time" src="https://user-images.githubusercontent.com/13599302/132640674-968fe144-9ff6-4eab-9b8f-9064c54a58a0.PNG">

blob properties don't have any attribute to enable "point_in_time_restore_for_containers" :
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_account

**Comment by aristosvo** [comment]
Hi @kumehta !

I believe this is a duplicate of #9020 and can be closed. I tried implementing it yesterday, but it is still waiting on upstream Microsoft fixes 😢 

See also:
- [My trial yesterday](https://github.com/hashicorp/terraform-provider-azurerm/pull/13267)
- [Upstrream issue](https://github.com/Azure/azure-rest-api-specs/issues/11237 )

**Comment by katbyte** [comment]
closing as duplicate of #9020
